dotcom-rendering/src/components/LatestLinks.importable.tsx

```diff
@@ -122,7 +122,7 @@ const extractAboutThreeLines = (text: string) =>
  *
  * ---
  *
- * (No visual story exists)
+ * [`LatestLinks` on Chromatic](https://www.chromatic.com/component?appId=63e251470cfbe61776b0ef19&csfId=components-latestlinks)
```

Comment:
This link will only work once this PR is merged. For now, use https://www.chromatic.com/component?appId=63e251470cfbe61776b0ef19&csfId=components-latestlinks&buildNumber=2884&k=64e1f8c175f50aaeb67d9109-320px-interactive-true&h=8&b=-1